frontend/src/api/team.js

```diff
@@ -478,6 +478,8 @@ const bulkDeviceMove = async (teamId, devices, moveTo, id = undefined) => {
     } else if (moveTo === 'unassigned') {
         data.instance = null
         data.application = null
+    } else if (moveTo === 'group') {
+        data.deviceGroup = id
```

Comment:
This is related to device moving a device into/out-of an instance or application (not group) as specified in the `@param` 

Why is this added?